solution/ui/e2e/cypress/support/common-commands/policyDocs.js

```diff
@@ -0,0 +1,49 @@
+export const checkPolicyDocs = ({ username, password, landingPage }) => {
+    cy.intercept("**/v3/content-search/?q=test**", {
```

Comment:
this is the utility method that will be used to verify that Internal and External policy docs are retrieved and displayed as expected.  This is currently used in the Search and Subjects test suites.